pallets/runtime/src/multisig.rs

```diff
@@ -409,7 +412,15 @@ impl<T: Trait> Module<T> {
             Error::<T>::AlreadyApproved
         );
         if let Some(proposal) = Self::proposals(&multisig_proposal) {
-            Self::charge_fee(multisig.clone(), proposal.get_dispatch_info().weight)?;
+            ensure!(
+                T::ChargeTxFeeTarget::charge_fee(
+                    signer,
+                    proposal.encode().len().try_into().unwrap_or_default(),
+                    proposal.get_dispatch_info(),
+                )
+                .is_ok(),
+                Error::<T>::FailedToChargeFee
+            );
```

Comment:
Wouldn't the fees will be deducted only if approvals >= approvals_needed?

Reply:
Nice catch, I'll fix it.